lib/configManager.js

```diff
@@ -48,8 +50,8 @@ module.exports = class ConfigManager {
    * @return The parsed YAML file
    */
   async loadGlobalSettingsYaml () {
-    const CONFIG_PATH = '.github'
-    const filePath = path.posix.join(CONFIG_PATH, 'settings.yml')
+    const CONFIG_PATH = env.CONFIG_PATH
```

Comment:
Have you tested this change with a different path outside of `.github` folder? As described in the probot docs https://probot.github.io/api/latest/classes/context.Context.html#config the configuration needs to live inside `.github` folder:

> Name of the YAML file in the .github directory

See as well https://probot.github.io/docs/best-practices/#store-configuration-in-the-repository

Reply:
Yes it worked.

```
SAFE_SETTINGS_ADMIN_REPO=.github
SAFE_SETTINGS_CONFIG_PATH=.config/
SAFE_SETTINGS_SETTINGS_FILE_PATH=safe-settings.yml
```

<img width="1059" alt="test" src="https://user-images.githubusercontent.com/616137/216501646-02e92eff-b055-442e-a202-2793c4486f86.png">

```
INFO (server): POST http://localhost:3000/ - 202
INFO (event): Updating repo with topics github,probot,new-topic,another-topic,topic-12
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): Updating repo with topics github,probot,new-topic,another-topic,topic-12
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): Updating repo with topics github,probot,new-topic,another-topic,topic-12
    id: "ece7c0c0-a36c-11ed-86f0-6d726ae5924e"
INFO (event): Updating repo with topics github,probot,new-topic,another-topic,topic-12
    id: "ece7c0c0-a36c-11ed-86f0-6d726ae5924e"
INFO (event): Updating repo with topics github,probot,new-topic,another-topic,topic-12
    id: "14336a30-a36d-11ed-955b-e30a62367fc8"
INFO (event): Updating repo with topics github,probot,new-topic,another-topic,topic-12
    id: "14336a30-a36d-11ed-955b-e30a62367fc8"
INFO (http): POST / 200 - 13ms
INFO (http): POST / 200 - 8ms
INFO (server): POST http://localhost:3000/ - 200
INFO (event): Finding teams for empirl-sandbox/test
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (http): POST / 200 - 9ms
INFO (http): POST / 200 - 7ms
INFO (server): POST http://localhost:3000/ - 200
INFO (event): Getting team with the parms {"name":"bots","permission":"push"}
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): Getting team with the parms {"name":"code-owners","permission":"admin"}
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): Repo test Passed Validation for pattern [a-zA-Z0-9_-]+
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): adding team bots to repo test
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): Finding teams for empirl-sandbox/test
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): adding team code-owners to repo test
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): Getting team with the parms {"name":"bots","permission":"push"}
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): Getting team with the parms {"name":"code-owners","permission":"admin"}
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): Repo test Passed Validation for pattern [a-zA-Z0-9_-]+
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): adding team code-owners to repo test
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): adding team bots to repo test
    id: "edd0c130-a36c-11ed-9add-603722a602c7"
INFO (event): Finding teams for empirl-sandbox/test
    id: "ece7c0c0-a36c-11ed-86f0-6d726ae5924e"
INFO (event): Getting team with the parms {"name":"bots","permission":"push"}
    id: "ece7c0c0-a36c-11ed-86f0-6d726ae5924e"
INFO (event): Getting team with the parms {"name":"code-owners","permission":"admin"}
    id: "ece7c0c0-a36c-11ed-86f0-6d726ae5924e"
INFO (event): Repo test Passed Validation for pattern [a-zA-Z0-9_-]+
    id: "ece7c0c0-a36c-11ed-86f0-6d726ae5924e"
...
```